ramls/dataExport.raml

```diff
@@ -68,80 +69,26 @@ resourceTypes:
         pageable,
         validate
       ]
-  /fileDefinitions:
-    displayName: File upload service
-    description: Service provides file uploading API
-    post:
-      description: Method to create file definition to use it for the file uploading
-      is: [validate]
-      body:
-        application/json:
-          type: fileDefinition
-      responses:
-        201:
-          body:
-            application/json:
-              type: fileDefinition
-        400:
-          description: "Bad request"
-          body:
-            text/plain:
-              example: "Bad request"
-        422:
-          description: "Unprocessable Entity"
-          body:
-            application/json:
-              type: errors
-        500:
-          description: "Internal server error"
-          body:
-            text/plain:
-              example: "Internal server error"
-    /{fileDefinitionId}:
-      get:
-        description: Method to get file definition by id
-        responses:
-          200:
-            body:
-              application/json:
-                type: fileDefinition
-          400:
-            description: "Bad request"
-            body:
-              text/plain:
-                example: "Bad request"
-          404:
-            description: "Not found"
-            body:
-              text/plain:
-                example: "Not found"
-          500:
-            description: "Internal server error"
-            body:
-              text/plain:
-                example: "Internal server error"
-      /upload:
-        post:
-          description: Method to upload file
-          body:
-              application/octet-stream:
-          responses:
-            200:
-              body:
-                application/json:
-                  type: fileDefinition
-            400:
-              description: "Bad request"
-              body:
-                text/plain:
-                  example: "Bad request"
-            404:
-              description: "Not found"
-              body:
-                text/plain:
-                  example: "Not found"
-            500:
-              description: "Internal server error"
-              body:
-                text/plain:
-                  example: "Internal server error"
+    /{jobId}/download/{exportFileId}:
```

Comment:
'jobId' sometimes confuses UI devs, they can not understand whether ' jobExecutionId 'or 'jobProfileId'. They have asked me 5 times :)  Maybe makes sense to rename to 'jobExecutionId'